index.ts

```diff
@@ -17,6 +17,8 @@ const QuickLRU = require('quick-lru');
 type Filter = (msg: Msg) => boolean;
 type IndexItem = [string, number, MsgId];
 
+let isBlocking = (obj:Object, cb:Function) => { cb(undefined, true) };
```

Comment:
Shouldn't this callback be `cb(undefined, false)`? Otherwise the default case will say that *all* posts are blocked